models/Review.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const ReviewSchema = new Schema(
+  {
+    versionId: { type: Schema.Types.ObjectId, ref: "ListVersion", required: true },
+    description: { type: Schema.Types.ObjectId, ref: "TextString", required: true },
```

Comment:
I think it's better to make this not required, so that the admin can create a review before he adds the actual description.
Alon, what do you think?